components/dashboard/src/components/Loader.tsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { ReactComponent as Spinner } from "../icons/Spinner.svg";
+
+export function SpinnerLoader(props: { content: string }) {
+    const content = props.content ?? "Fetching data...";
```

Comment:
**suggestion**: Being more specific here sounds better, right? I've already documented the following guidelines for the button component. Sharing for visibility:

> Use concise language that conveys what happens when the button is activated.

```suggestion
    const content = props.content ?? "Fetching access tokens...";
```

Reply:
This is a common component, so it should be `data` as a placeholder.

PR's still WIP 🙈

Reply:
> This is a common component, so it should be data as a placeholder.

Oh, I just realized this is a new separate component! Thanks for the hint, @mustard-mh! 🐈

Although there's no right or wrong approach, allow me to disagree with the general approach though, @mustard-mh! Fetching data sounds quite vague that does not provide much help to users besides looking a bit scary.

What do you think of going with a less technical term here instead, then?

```suggestion
    const content = props.content ?? "Loading...";
```

> PR's still WIP

@mustard-mh I hope this review didn't come across as additional pressure! Cc @jeanp413 

[Easy pace—it's the winning pace!](https://gitpod.slack.com/archives/C01KGM9FXB8/p1668783628316019)

Reply:
❓**question**: Also, since we're here, can this component simply show the spinner with no text (content)?

Reply:
>❓question: Also, since we're here, can this component simply show the spinner with no text (content)?

Pass content as empty string will make it disappear, I will change it to default not show `content` property 🙏

Reply:
> I will change it to default not show content property 

:100: